SU-SWS/stanford-jumpstart-deployer
# Update stanford_private_page to 2.x branch or release

# READY FOR REVIEW

# Summary
- Security update - update `stanford_private_page` to `7.x-2.x` on development and `7.x-2.0-alpha3` on production

# Needed By (Date)
- Soon

# Criticality
- How critical is this PR on a 1-10 scale? 4
- Only affects products that use stanford_private_page

# Steps to Test

1. See what Travis does
2. Test building JSLab, JSE, JSVPSA development and ensure that they build
2. Test building JSLab, JSE production and ensure that they build

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? JSE, JSL, JSVPSA

# Associated Issues and/or People
- JIRA ticket (?)
- Anyone who should be notified? (@pookmish @boznik @cjwest )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
